### DISCUSSION TITLE
年终总结：新的技术周期，每个前端都有更好的职业转型机遇 | J实验室

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/new-blog#10] [General]
# 年终总结：新的技术周期，每个前端都有更好的职业转型机遇 | J实验室

AI引爆新的技术周期，前端工程师该如何享受这个周期的技术红利、找到自己在这个时代的技术转型机遇？

https://weijunext.com/article/2023-summary-of-front-end-and-ai

**Comment by lichenbuliren** [comment]
受用了，感谢分享
